src/i18n/ja.json

```diff
@@ -0,0 +1,73 @@
+{
+  "1 reply": "1件の返信",
+  "Attach files": "ファイルを添付する",
+  "Cancel": "キャンセル",
+  "Channel Missing": "チャネルがありません",
+  "Close": "選ぶ",
+  "Close emoji picker": "絵文字ピッカーを閉じる",
+  "Commands matching": "一致するコマンド",
+  "Connection failure, reconnecting now...": "接続に失敗しました。今すぐ再接続してください...",
+  "Delete": "消去",
+  "Delivered": "配信済み",
+  "Edit Message": "メッセージの編集",
+  "Edit message request failed": "メッセージの編集要求が失敗しました",
+  "Emoji matching": "絵文字マッチング",
+  "Empty message...": "空のメッセージ...",
+  "Error adding flag": "フラグの追加中にエラーが発生しました",
+  "Error connecting to chat, refresh the page to try again.": "チャットへの接続中にエラーが発生しました。ページを更新して再試行してください。",
+  "Error deleting message": "メッセージの削除中にエラーが発生しました",
+  "Error muting a user ...": "ユーザーのミュート中にエラーが発生しました...",
+  "Error pinning message": "メッセージの固定エラー",
+  "Error removing message pin": "メッセージピンの削除中にエラーが発生しました",
+  "Error unmuting a user ...": "ユーザーのミュート解除中にエラーが発生しました...",
+  "Error uploading file": "ファイルのアップロード中にエラーが発生しました",
+  "Error uploading image": "画像のアップロード中にエラーが発生しました",
+  "Error · Unsent": "エラー・未送信",
+  "Error: {{ errorMessage }}": "エラー: {{ errorMessage }}",
+  "Flag": "国旗",
+  "Message Failed · Click to try again": "メッセージが失敗しました · クリックして再試行してください",
+  "Message Failed · Unauthorized": "メッセージが失敗しました · 許可されていない",
+  "Message deleted": "メッセージが削除されました",
+  "Message has been successfully flagged": "メッセージに正常にフラグが付けられました",
+  "Message pinned": "固定されたメッセージ",
+  "Mute": "無音",
+  "New": "新しい",
+  "New Messages!": "新しいメッセージ!",
+  "No results found": "結果が見つかりません",
+  "Nothing yet...": "まだ何もありません...",
+  "Only visible to you": "あなただけに見える",
+  "Open emoji picker": "絵文字ピッカーを開く",
+  "People matching": "一致する人",
+  "Pick your emoji": "絵文字を選んでください",
+  "Pin": "ピン",
+  "Pinned by": "によって固定",
+  "Reply": "返事",
+  "Reply to Message": "メッセージに返信",
+  "Search": "探す",
+  "Searching...": "検索中...",
+  "Send": "送信",
+  "Send message request failed": "メッセージ送信リクエストが失敗しました",
+  "Sending...": "送信...",
+  "Slow Mode ON": "スローモードオン",
+  "Start of a new thread": "新しいスレッドの開始",
+  "This message was deleted...": "このメッセージは削除されました...",
+  "Thread": "糸",
+  "Type your message": "メッセージを入力してください",
+  "Unmute": "ミュートを解除する",
+  "Unpin": "固定を解除する",
+  "Wait until all attachments have uploaded": "すべての添付ファイルがアップロードされるまで待ちます",
+  "You have no channels currently": "現在チャンネルはありません",
+  "You've reached the maximum number of files": "ファイルの最大数に達しました",
+  "live": "住む",
+  "this content could not be displayed": "このコンテンツは表示できませんでした",
+  "{{ commaSeparatedUsers }} and {{ moreCount }} more": "{{ commaSeparatedUsers }} と {{moreCount}} もっと",
```

Comment:
not 100% sure how this translation magic works but might need the surrounding spaces here IDK